src/lib.cairo

```diff
@@ -16,7 +19,9 @@ trait IVerifier<TContractState> {
 trait IStarknetID<TContractState> {
     fn owner_of(self: @TContractState, token_id: felt252) -> starknet::ContractAddress;
 
-    fn set_verifier_data(self: @TContractState, token_id: felt252, field: felt252, data: felt252);
+    fn set_verifier_data(
+        self: @TContractState, token_id: felt252, field: felt252, data: felt252, domain: u32
```

Comment:
shouldn't we use `u128` for `token_id` to have the same type as in the identity contract ?

Reply:
right